lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -372,6 +377,10 @@ class TextEditingElement {
     if (inputConfig.obscureText) {
       domElement.setAttribute('type', 'password');
     }
+    if (browserEngine == BrowserEngine.webkit && inputConfig.autocorrect) {
```

Comment:
Two comments:
1. Why not set the autocorrect attribute in all browsers? Even if it doesn't do anything now, there's no harm in setting it. And you'll be able to write a test for it :)
2. Because the condition has `inputConfig.autocorrect`, the "off" string below will never be used.

Reply:
1. Sure, I can do it. (It is not my preferred way though. I would prefer to add something when I know it works.)
2. thanks, I missed it :)

Reply:
1. I don't have a strong preference either way, so feel free to keep the webkit check. But we need a test for this so it doesn't regress over time. Consider looking at https://github.com/flutter/engine/pull/13044 for ideas on how to override `browserEngine` for testing purposes.

Reply:
I added it for all the browsers, added unit tests. I also added one line of comment for future developers who wants to test the feature manually. Thanks!